packages/runtime/src/stratus.ts

```diff
@@ -2417,29 +2197,6 @@ Stratus.LocalStorage.Listen('stratus-core', (data: any) => {
 })
 // localStorage.setItem('stratus-core', 'foo')
 
-// DOM Listeners
-// -------------
-
-// This function executes when the DOM is Ready, which means
-// the DOM is fully parsed, but still loading sub-resources
-// (CSS, Images, Frames, etc).
-Stratus.DOM.ready = (fn: any) => {
```

Comment:
Removing these functions will break any old code referencing these functions.  For the time being, we can do something like:

```ts
import {ready} from '@stratusjs/core/dom'

/**
 * @deprecated use `ready()` from `core/dom` instead
 */
Stratus.DOM.ready = ready

Reply:
Stratus.DOM has been replaced with dom.tsc.
so these functions now/still exist but are managed outside this file:
- Stratus.DOM.ready
- Stratus.DOM.complete
- Stratus.DOM.unload
- Stratus.DOM.Select
- Stratus.DOM.Selector
- Stratus.Select
- Stratus.Selector

Reply:
https://github.com/Sitetheory/stratus/pull/652/files#diff-6c72bd323aae7aff5417704e02058230c65a9fd01d0a59b81d3ed4c299c8cd17L205-R208

Reply:
Oh I see!  Awesome!